test/e2e/reboot.go

```diff
@@ -160,13 +168,24 @@ func rebootNode(c *client.Client, provider, name string, result chan bool) {
 		result <- false
 		return
 	}
-	Logf("Calling reboot on %s", name)
-	rebootCmd := "sudo reboot"
+	var rebootCmd string
+	if clean {
```

Comment:
See comment above.  I think that this needs to be more flexible than a boolean option.